### PR TITLE
data: WillBeebe.json

### PR DESCRIPTION
**Pull request by WillBeebe**
+ simplify bio
+ yeet => buy

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7707"></a>

### CODE REVIEW

File: data/WillBeebe.json [hunk 1]
```diff
@@ -36,7 +35,7 @@
       "icon": "FaLinkedin"
     },
     {
```
Comment (CBID2) [inline data/WillBeebe.json:37]: [Proofreading] This section is missing the url and name fields for LinkedIn. Check out the [quickstart on Linkfree's website to learn how to add this information](https://linkfree.io/docs/quickstart). 
```suggestion
      "icon": "FaLinkedin"
    },
    {
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @WillBeebe! :) Your profile is off to a great start! :) I just gave a suggestion on how to improve it.

File: data/WillBeebe.json (hunk 1 above)
Comment (SaraJaoude) [reply data/WillBeebe.json:37]: @CBID2 there have not been changes to this line and the LinkedIn link object has a URL...

```js
     {
      "name": "Let's connect on LinkedIn",
      "url": "https://www.linkedin.com/in/will-beebe/",
       "icon": "FaLinkedin"
     },
```

Therefore this is incorrect and I will mark as resolved.

**Review by SaraJaoude — APPROVED** [review]
@WillBeebe the suggestion was incorrect so I marked as resolved - your PR looks good and I will merge this now

**Pull request merged**